app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -344,20 +355,54 @@ export const FormAudience = ({
           </LinkExternal>
         </p>
 
-        <Form.Row>
-          <Form.Group as={Col} className="mx-5" controlId="populationPercent">
-            <Form.Label>Percent of clients</Form.Label>
+        <Form.Row data-testid="population-percent-top-row">
+          <Form.Group
+            as={Col}
+            className="mt-2 mb-0"
+            controlId="populationPercent"
+            data-testid="population-percent-top"
+          >
+            <InputGroup className="mx-0 d-flex">
+              <Form.Label
+                style={{
+                  display: "inline-block",
+                  marginBottom: 0,
+                }}
+              >
+                Percent of clients
+              </Form.Label>
+            </InputGroup>
             <InputGroup>
               <Form.Control
                 {...formControlAttrs(
                   "populationPercent",
                   POSITIVE_NUMBER_FIELD,
                 )}
+                type="hidden"
+                value={populationPercent}
+              />
+              <Form.Control
+                aria-describedby="populationPercent-unit"
+                type="range"
+                min="0"
+                max="100"
+                step="5"
```

Comment:
Why step: 5?

Reply:
It gives the user an easier way to land on _exactly_ the percent they want, which would usually be like 20%, 50%, etc. If they want something uber specific like 8%, they can type it in. It's a little annoying to use the slider to try to fine tune. I'm fine with changing it to 1's though, what do you think?

Reply:
Yeah I mean it's hard to guess what users will want and is staggeringly trivial to change so let's try 5 and see how users like it 👍

Reply:
Yeah, looks like out of the current Live ones there's only _one_ experiment with a non-5 popultion: 
<img width="391" alt="image" src="https://user-images.githubusercontent.com/43795363/203131206-26865af1-b436-4e2a-8460-c533a86d3817.png">

Reply:
Using data to support a technical decision earns you The Duck Of Data Driven Decisions:

 
![tenor](https://user-images.githubusercontent.com/119884/203131613-78a76087-c58d-413c-a363-6eeef63d6e1e.gif)

Reply:
🦆🦆🦆🦆🦆